apps/challenges/views.py

```diff
@@ -707,8 +707,7 @@ def get_all_submissions_of_challenge(request, challenge_pk, challenge_phase_pk):
 
         # Filter submissions on the basis of challenge for host for now. Later on, the support for query
         # parameters like challenge phase, date is to be added.
-
-        submissions = Submission.objects.filter(challenge_phase__challenge=challenge).order_by('-submitted_at')
+        submissions = Submission.objects.filter(challenge_phase=challenge_phase).order_by('-submitted_at')
```

Comment:
@isht3 Please add support for filtering on the basis of `challenge_phase` corresponding to a `challenge`.

Reply:
@RishabhJain2018 On line 700 `challenge_phase = ChallengePhase.objects.get(pk=challenge_phase_pk, challenge=challenge)` and I used this `challenge_phase` to obtain its submissions. This `challenge_phase` is of that particular challenge, right?

Reply:
@isht3 Yes you are right! My bad, I didn't see that line during review.

Reply:
@RishabhJain2018 Totally cool! Ill start with the other issue. :smile: